spec/nexpose/site_spec.rb

```diff
@@ -58,4 +58,77 @@
       end
     end
   end
+
+  context 'with assets' do
+    before do
+      subject.assets = [
+        Nexpose::IPRange.new('192.168.1.1'),
+        Nexpose::IPRange.new('192.168.1.0', '192.168.1.255'),
+        Nexpose::IPRange.new('192.168.1.1/24'),
+        Nexpose::HostName.new('nexpose.local')
+      ]
+    end
+
+    describe '#remove_asset' do
+      context 'with a hostname' do
+        it 'removes a new HostName from the assets list' do
+          subject.remove_asset('nexpose.local')
+
+          expect(subject.assets)
+            .to_not include(Nexpose::HostName.new('nexpose.local'))
+        end
+      end
+
+      context 'with a single IP address' do
+        it 'removes a new IPRange from the assets list' do
+          subject.remove_asset('192.168.1.1')
+
+          expect(subject.assets)
+            .to_not include(Nexpose::IPRange.new('192.168.1.1'))
+        end
+      end
+
+      context 'with a multiple IP address' do
+        # TODO: The IPRange class apparently doesn't handle CIDR notation on
+        # the client side.
+        xit 'removes a new IPRange from the assets list' do
+          subject.remove_asset('192.168.1.0/24')
+
+          expect(subject.assets)
+            .to_not include(Nexpose::IPRange.new('192.168.1.0', '192.168.1.255'))
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.